readme.md

```diff
@@ -0,0 +1,30 @@
+# Super Secret Puzzle Tools to Win
+
+## Tools to Create
+(Sign up for whatever you decide to work on here so we don't duplicate work)
+
+Basically what I want is all the random online tools in one easy lightweight place or in script version.
+
+- Word Ladder tools
+    - "Next Word" (adjacent) word generator
+    - Ladder solver (or link to a ladder solver)
+    - [Example](http://ceptimus.co.uk/wordladder.php)
+- Caesar Cipher tools
+    - Shift letters by independent numbers
+    - Shift phrase by number
+    - Shift phrase brute force
+    - Or just use [this](https://www.dcode.fr/shift-cipher)
+- Morse Code Translator
+    - Encode
+    - Decode
+    - [Example](https://morsecode.scphillips.com/translator.html)
+- Improvements to wordhelper
+    - Starts with
+    - Ends with
```

Comment:
Starts with, ends with and contains are supported through regex. Are you suggesting a shortcut for these functionalities?